app/controllers/articles_controller.rb

```diff
@@ -1,10 +1,11 @@
 class ArticlesController < ApplicationController
+  before_action :set_article, only: [:show, :edit, :update, :destroy ]
+
```

Comment:
Hi @tsheporamantso,
Perfect job on keeping your code base DRY by defining this ``set_article method` 🤝